src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -1201,5 +1201,15 @@ public static ConditionalAccessExpressionSyntax GetParentConditionalAccessExpres
 
             return null;
         }
+
+        public static ConditionalAccessExpressionSyntax GetInnerMostConditionalAccessExpression(this SyntaxNode node)
+        {
+            if (!(node is ConditionalAccessExpressionSyntax))
+            {
+                return null;
+            }
+
+            return node.DescendantNodesAndSelf().OfType<ConditionalAccessExpressionSyntax>().Last();
```

Comment:
Thanks for the note @Pilchie. I'll look out for this in the future. To think that I hand wrote what i needed and then swapped it for a 1 line linq statement that isn't the same just before i sent the PR :anguished: